.drone.yml

```diff
@@ -1760,9 +1760,12 @@ steps:
 
   # compare indexing from FROM_BLOCK_HEADER to TO_BLOCK_HEADER between defined nodes
   - name: indexer-test
-    image: simplestakingcom/tezos-node-bootstrap:latest
+    image: simplestakingcom/tezos-node-bootstrap:test
     commands:
-      - tezos-node-bootstrap --indexer-test --level=$${TO_BLOCK_HEADER} --node_tezedge=$${TEZEDGE_NODE_RPC_CONTEXT_ROOT} --indexer_tezedge=$${TEZEDGE_NODE_INDEXER_ROOT} --node_ocaml=$${OCAML_NODE_RPC_CONTEXT_ROOT_1} --indexer_ocaml=$${OCAML_NODE_INDEXER_ROOT}
+      - tezos-node-bootstrap indexer-test --level=$${TO_BLOCK_HEADER} --tezedge-new-node=$${TEZEDGE_NODE_RPC_CONTEXT_ROOT} --tezedge_indexer=$${TEZEDGE_NODE_INDEXER_ROOT} --ocaml-node=$${OCAML_NODE_RPC_CONTEXT_ROOT_1} --indexer_ocaml=$${OCAML_NODE_INDEXER_ROOT}
+    # TODO: TE-349 - if new docker image is published (remove failure and detached)
```

Comment:
@adonagy 
I guess, this is leftover after bad rebase :),
this lines were removed in develop already:
```
# TODO: TE-349 - if new docker image is published (remove failure and detached)
    failure: ignore
    detach: true
```